scripts/dts/python-devicetree/tests/test_dtlib.py

```diff
@@ -2555,3 +2584,24 @@ def test_filename_and_lineno():
         assert prop1A.lineno == 4
         assert os.path.samefile(prop1B.filename, main_file)
         assert prop1B.lineno == 13
+
+        # Test the generated file comments are correct
+        assert str(dt) == '''
+/dts-v1/;
+
+/* node '/' defined in included.dtsi:2 */
+/ {
+
+	/* node '/test-node1' defined in included.dtsi:3 */
+	node1: test-node1 {
+		prop1A = "value1A"; /* in included.dtsi:4 */
+		prop1B = "value1B"; /* in test_with_include.dts:13 */
```

Comment:
on the topic of things that are embarassingly simple to add, can you update the test so that there are string values of different length so that that alignment of the comments can be tested as well?